### ISSUE TITLE
[FEATURE] Export databases as nfo

### ISSUE THREAD

**Issue by shtrom** [robweber/xbmcbackup#219]
Not a bug, but a feature request / suggestion.

Kodi has an export mechanism where it is able to dump the libraries as either a single directory, or a number of per-media-entry nfo files. The latter can be re-imported simply by rescanning the media directories.

There has been a few questions over the years about how to schedule such exports (e.g., https://forum.kodi.tv/showthread.php?tid=231294, https://www.reddit.com/r/kodi/comments/60mde3/automatic_library_export/, https://forum.kodi.tv/showthread.php?tid=104611, ...). This is also what led me to installing the addon, as the description on the [Kodi wiki says that it can “export your database”](https://kodi.wiki/view/Add-on:Backup) (which it does, but not the way Kodi Library export does it).

It would be useful if, in addition to the other backup mechanisms, the addon were able to also trigger Kodi Library exports (single or multi files).

**Comment by robweber** [comment]
Good idea. I see there is a JSON command for doing both exports of the [Audio](https://kodi.wiki/view/JSON-RPC_API/v13#AudioLibrary.Export) and [Video](https://kodi.wiki/view/JSON-RPC_API/v13#VideoLibrary.Export) Library. These could be leaned on to do the export.
